bom/pom.xml

```diff
@@ -119,6 +119,11 @@
         <!-- We MUST override odlparent's too old protobuf version here for jetcd -->
         <version>3.4.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.ops4j.pax.cdi</groupId>
+        <artifactId>pax-cdi-api</artifactId>
```

Comment:
not sure why this is suddenly needed, but it can't really hurt either, so fine with me

Reply:
it's because of https://git.opendaylight.org/gerrit/#/c/75762/ (of all people, I should remember!)